src/osd/scrubber/pg_scrubber.cc

```diff
@@ -656,6 +656,77 @@ Scrub::sched_conf_t PgScrubber::populate_config_params() const
 }
 
 
+// handling Asok's "scrub" & "deep_scrub" commands
+
+namespace {
+void asok_response_section(
+    ceph::Formatter* f,
+    bool is_periodic,
+    scrub_level_t scrub_level
+    /*const char* section_value*/)
+{
+  f->open_object_section("result");
+  f->dump_bool("deep", (scrub_level == scrub_level_t::deep));
+  f->dump_bool("must", !is_periodic);
+  f->close_section();
```

Comment:
We stopped dumpig the `stamp` field though the information in the period variant is still available. Is this intentional?

Reply:
@rzarzynski - I am redoing this whole thing:

- the scrub and deep-scrub commands will have no parameters, and will always force a scrub

- the test/debug behavior, that is used extensively by tests and during debug, will have its own command.

No need to continue reviewing this version.

Reply:
Re the "stamp" - a good point. I have removed it as it only exists in one situation.
Will restore, just for the case that there was someone using that.

Reply:
Sure, please let me know after it's ready for another round.

Reply:
@rzarzynski  - it is now, I hope. The comments are from before I changed the PR into a draft, modified it, run some Teu tests, and now marked it Ready again.

Reply:
(and I have restored the 'stamp' field, as you've suggested)